services/provider/proto/provider.proto

```diff
@@ -90,6 +93,12 @@ message StorageConfigResponse{
     repeated ExternalResource externalResource = 1;
     // Contains hash of desired config
     string desiredConfigHash = 2;
+
+    message StorageStatus {
+      string FailoverMaintenanceMode = 1;
```

Comment:
As agreed in our discussion, we should not use Failover or any specific reason for the MM at this point. 
I would say this need to be something like `bool SystemInMaintannceMode` or something similar